changelog/fix-major-version-check

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fixed version check regex
```

Comment:
Just noticed it's not an insignificant changelog, and it will appear in the next release's changelog if it stays like that.

Could you redo it so it's marked as a "comment"? It should, in the end, be written as:

> Comment: Fixed version check regex

Reply:
Fixed with ad04862fcc10ea2e204dce88f7dfb8f248d5c7b8